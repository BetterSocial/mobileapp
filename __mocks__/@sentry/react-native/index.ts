export const addBreadcrumb = jest.fn();
