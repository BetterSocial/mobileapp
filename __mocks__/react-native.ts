import * as RN from 'react-native';

RN.Animated.timing = () => ({
    start: () => jest.fn()
})

module.exports = RN