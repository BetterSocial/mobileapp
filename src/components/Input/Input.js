import * as React from 'react';
import {Dimensions, StyleSheet, TextInput} from 'react-native';

const {width} = Dimensions.get('screen');

const Input = ({...props}) => <TextInput style={styles.input} {...props} />;

export default Input;

const styles = StyleSheet.create({
  input: {
    borderWidth: 1,
    borderRadius: 8,
    borderColor: '#BDBDBD',
    paddingHorizontal: 23,
    paddingVertical: 13,
    width: width - 100,
  },
});
