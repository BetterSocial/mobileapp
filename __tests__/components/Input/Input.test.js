import * as React from 'react';
import { render, cleanup } from '@testing-library/react-native';
import Input from '../../../src/components/Input/Input';


describe('Input should run correctly', () => {


    afterEach(cleanup)
    it('should match snapshot', () => {
        const {toJSON} = render(<Input  />)
        expect(toJSON).toMatchSnapshot()
    })
})