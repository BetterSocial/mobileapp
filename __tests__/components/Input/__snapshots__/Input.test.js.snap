// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should run correctly should match snapshot 1`] = `[Function]`;
